### PR TITLE
Allow setting the Slack channel from ENV

### PR DESCRIPTION
**Pull request by cdmwebs**
Works on my machine! Code reviews gladly accepted :)

### CODE REVIEW

File: app.json [hunk 1]
```diff
@@ -11,6 +11,10 @@
       "description": "A Slack API token for your account",
       "value": "SLACK API TOKEN GOES HERE"
     },
+    "SLACK_CHANNEL": {
+      "description": "The Slack channel you'd like your notables to appear in",
```
Comment (cdmwebs) [inline app.json]: Grammar!

File: app.json [hunk 2]
```diff
@@ -11,6 +11,10 @@
       "description": "A Slack API token for your account",
       "value": "SLACK API TOKEN GOES HERE"
     },
+    "SLACK_CHANNEL": {
+      "description": "The Slack channel in which you'd like your notables to appear",
+      "value": "SLACK CHANNEL NAME HERE"
```
Comment (pusewicz) [inline app.json]: I'd put "SLACK CHANNEL NAME GOES HERE" just to make it consistent with the other values! :smile:

**Comment by jasondew** [comment]
@cdmwebs Awesome, this looks great! I agree on the copy change but other than that :+1:

**Comment by cdmwebs** [comment]
Done! @jasondew @pusewicz

**Comment by jasondew** [comment]
@cdmwebs Thanks man!

**Comment by cdmwebs** [comment]
Sure thing! FYI: I didn't update the docs branch.

**Pull request merged**